spigot/src/main/java/com/convallyria/forcepack/spigot/ForcePackSpigot.java

```diff
@@ -69,6 +69,8 @@ public final class ForcePackSpigot extends JavaPlugin implements ForcePackAPI {
 
     private BukkitAudiences adventure;
 
+    public final Set<UUID> temporaryExemptedPlayers = new HashSet<>();
```

Comment:
If a player simply does not send a resource pack response (e.g. cheats), they will stay in the set. We should remove the player from the set when they quit.